### ISSUE TITLE
Buttons #5-10 no longer work 

### ISSUE THREAD

**Issue by funkytek** [codrops/DistortedButtonEffects#5]
There are no effect on mouseover or click for button examples #5-10.  #1-4 still work as expected.  Please advise?

**Comment by botelho** [comment]
Seems to work again!! :) Do you confirm?

**Comment by byhnme** [comment]
Distorted button effect number 8 and 9 not work [Distorted button](https://codepen.io/tomo_yuen/pen/RpKMzN)

**Comment by lamhapyan** [comment]
您的邮件我已收到，并尽快回复你，请耐心等待!

**Comment by crnacura** [comment]
The button effects still work in the demo: https://tympanus.net/Development/DistortedButtonEffects/
Anything different in the code you use in your version @byhnd ?

**Comment by byhnme** [comment]
Yes, that's why I'm confused. When implemented on my web it doesn't work. I have changed the image address "xlink:href="img/ripple.png" to "https://tympanus.net/Development/DistortedButtonEffects/img/ripple.png"

But it still doesn't work.

@crnacura
